README.md

```diff
@@ -15,7 +15,9 @@ To learn more about Pravega, visit https://pravega.io
 
 ## Prerequisites
 
-- Java 8+
+- Java 11+
+
+The `client` project is expected to be built into *Java 8* bytecode.
```

Comment:
I would say something like that:
In spite of the requirements of using JDK11+ to build this project, Pravega client artifacts are expected to be compatible with Java 8 runtime, only the server side components need JDK11+ to run.